docs/docs/rest-api/public/api/v2/types/volumeMount.raml

```diff
@@ -0,0 +1,62 @@
+#%RAML 1.0 DataType
```

Comment:
Agreed, I may have mixed pod and container volumes together by accident. It was long and kind of confusing in v3